app/controllers/api/community_solutions_controller.rb

```diff
@@ -15,7 +15,10 @@ def use_exercise
     end
 
     def search_params
-      params.permit(AssembleExerciseCommunitySolutionsList.keys)
+      params.permit(AssembleExerciseCommunitySolutionsList.keys).tap do |p|
+        # Filter on passed head tests by default
+        p[:passed_head_tests] = "true" if p[:passed_head_tests].nil?
+      end
```

Comment:
What's the there a reason to have this here rather than having the default in the assembler?